Framework/PythonInterface/test/python/plugins/algorithms/LoadLampTest.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import (absolute_import, division, print_function)
+from mantid.simpleapi import config, mtd, LoadLamp
+import unittest
+
+
+class LoadLampTest(unittest.TestCase):
+
+    def setUp(self):
+        config.appendDataSearchSubDir('ILL/LAMP/')
+
+    def tearDown(self):
+        mtd.clear()
+
+    def test_1D(self):
```

Comment:
Could we test a few of the values we read from the file as well, just to be sure that they are being read in correctly? This could also apply to `test_2D_ragged` and `test_2D_tile`

Reply:
👍